examples/agents/elasticsearch.ts

```diff
@@ -0,0 +1,60 @@
+import "dotenv/config.js";
+import { BeeAgent } from "bee-agent-framework/agents/bee/agent";
+import { OpenAIChatLLM } from "bee-agent-framework/adapters/openai/chat";
+import { ElasticSearchTool } from "bee-agent-framework/tools/database/elasticsearch";
+import { FrameworkError } from "bee-agent-framework/errors";
+import { UnconstrainedMemory } from "bee-agent-framework/memory/unconstrainedMemory";
+
+const llm = new OpenAIChatLLM({
+  parameters: {
+    temperature: 0,
+  },
```

Comment:
I believe that 0 is a default value in our implementation.

Reply:
No default for openAI temperature

Reply:
Good catch. Changed in https://github.com/i-am-bee/bee-agent-framework/commit/ea8480850f1b1f84e9551e68d9165d1399909c5e